### PR TITLE
(Initial) Release v2.9.7+rev1

### PR DESCRIPTION
**Pull request by telphan**

### CODE REVIEW

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by telphan** [comment]
@resin-jenkins retest this please

**Comment by floion** [comment]
@resin-jenkins okay to test

File: layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend [hunk 1]
```diff
@@ -0,0 +1 @@
+FILES_COMPRESS=""
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend]: How is this broken? I remember this being fixed a while ago: https://github.com/resin-os/meta-resin/commit/1d7061075e28c15572cda65a7c841eaadb92ab55

Did you have any problems with this?

File: layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend [hunk 2]
```diff
@@ -0,0 +1 @@
+FILES_COMPRESS=""
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend]: How is this broken? I remember this being fixed a while ago: resin-os/meta-resin@1d70610

Did you have any problems with this?

Comment (telphan) [reply layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend]: I remember even after that commit the TX2 had the mentioned issue. I will give it a test.

Comment (telphan) [reply layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend]: @floion this is still broken

File: layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend (hunk 1 above)
Comment (telphan) [reply layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend]: Still broken

File: layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend (hunk 2 above)
Comment (agherzan) [reply layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend]: @telphan I remember we created an issue at upx about this? Would you reference it and maybe follow up with what they were requesting? I think it was a minimum test case.

Comment (agherzan) [reply layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend]: Uh wait. I found the issue: https://github.com/upx/upx/issues/130#issuecomment-338521057 . So we should mention this in the bbappend and make sure we remove it when the next version is released.

Comment (telphan) [reply layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend]: @agherzan I cannot find the issue. Maybe we have not raised it after all.

Comment (telphan) [reply layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend]: Included the issue in the bbappend.

File: layers/meta-resin-jetson/recipes-bsp/u-boot/u-boot-tegra/0001-Integrate-resin-u-boot.patch [hunk 3]
```diff
@@ -0,0 +1,40 @@
+From fc0ab962cd2315f330a5b619ec6b1cbb85f5141e Mon Sep 17 00:00:00 2001
+From: Theodor Gherzan <theodor@resin.io>
+Date: Mon, 9 Oct 2017 19:24:51 +0200
+Subject: [PATCH] Integrate resin-u-boot
+
+Signed-off-by: Theodor Gherzan <theodor@resin.io>
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-bsp/u-boot/u-boot-tegra/0001-Integrate-resin-u-boot.patch]: Missing Upstream-status

File: layers/meta-resin-jetson/recipes-core/packagegroups/packagegroup-resin.bbappend [hunk 4]
```diff
@@ -0,0 +1 @@
+RDEPENDS_${PN} += "tegra-firmware-xusb"
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-core/packagegroups/packagegroup-resin.bbappend]: Please add some details in the git log on why we need to add this

**Comment by floion** [comment]
@resin-jenkins okay to test

File: jetson-tx1.coffee [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+		version: 'yocto-pyro'
+		deployArtifact: 'resin-image-flasher-jetson-tx1.resinos-img'
+		compressed: true
+
+	options: [ networkOptions.group ]
+
+	configuration:
+		config:
+			partition:
+				primary: 10
```
Comment (floion) [inline jetson-tx1.coffee]: should this be changed like it was done for the TX2?

Comment (telphan) [reply jetson-tx1.coffee]: This will be part of the move to 28.2. When this board is updated.

**Pull request merged**